### ISSUE TITLE
Inverted/Incorrect IK?

### ISSUE THREAD

**Issue by TigerHix** [sotanmochi/HandPoseTransferForOculusQuest#1]
![image](https://user-images.githubusercontent.com/3406505/79315665-43dc5800-7ed1-11ea-96e3-76740a29e8c9.png)

Using latest Oculus Integration package (15.0) and Unity 2019.3.2f1, the sample project doesn't seem to work correctly - the hand bones look "inverted." Not sure if this is a problem of HandPoseTransfer or XRMecanimIKPlus.

**Comment by sotanmochi** [comment]
I'm sorry for the late response. 
I have fixed error and checked result using Unity 2018.4.21f1 and Oculus integration 16.0.
The orientation of hand joints had been changed due to updating Oculus Integration. 

Visualized hand joints for Oculus Integration 16.0
![OVRHandBoneVisualizer_16 0](https://user-images.githubusercontent.com/16936357/81405085-f5d60f80-9171-11ea-88c8-de72fc38b0d6.jpg)

Visualized hand joints for Oculus Integration 12.0
![OVRHandBoneVisualizer_12 0](https://user-images.githubusercontent.com/16936357/81405095-f9699680-9171-11ea-9000-2a0a4a36f2c2.jpg)
